tools/ci_connector_ops/ci_connector_ops/qa_engine/connector_adoption.sql

```diff
@@ -0,0 +1,46 @@
+WITH official_connector_syncs AS (
```

Comment:
Looks good to me, but yes - check with @alex-gron :D
... perhaps you also want to add the OSS information, in case we ever need it?

Reply:
Here's the query in [metabase](https://airbyte.metabaseapp.com/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJXSVRIIG9mZmljaWFsX2Nvbm5lY3Rvcl9zeW5jcyBBUyAoXG4gICAgU0VMRUNUICogRlJPTSBhaXJieXRlX3dhcmVob3VzZS5jb25uZWN0b3Jfc3luYyBXSEVSRSBpc19vZmZpY2lhbGx5X3B1Ymxpc2hlZCBhbmQgKGpvYl9zdGF0dXMgPSBcImZhaWxlZFwiIG9yIGpvYl9zdGF0dXMgPSBcInN1Y2NlZWRlZFwiKSBcbiksXG5hZG9wdGlvbl9wZXJfdmVyc2lvbiBBUyAoXG4gIFNFTEVDVFxuICAgIGNvbm5lY3Rvcl9kZWZpbml0aW9uX2lkLFxuICAgIGRvY2tlcl9yZXBvc2l0b3J5LFxuICAgIGNvbm5lY3Rvcl92ZXJzaW9uLFxuICAgIENPVU5UKGRpc3RpbmN0KHVzZXJfaWQpKSBhcyBudW1iZXJfb2ZfdXNlcnMsXG4gICAgQ09VTlQoZGlzdGluY3QoY29ubmVjdGlvbl9pZCkpIGFzIG51bWJlcl9vZl9jb25uZWN0aW9uc1xuICBGUk9NXG4gICAgb2ZmaWNpYWxfY29ubmVjdG9yX3N5bmNzXG4gIEdST1VQIEJZIGNvbm5lY3Rvcl9kZWZpbml0aW9uX2lkLCBkb2NrZXJfcmVwb3NpdG9yeSwgY29ubmVjdG9yX3ZlcnNpb25cbiksXG5qb2Jfc3RhdHVzX3Blcl92ZXJzaW9uIEFTIChcbiAgICBTRUxFQ1QgXG4gICAgICAgIGNvbm5lY3Rvcl9kZWZpbml0aW9uX2lkLFxuICAgICAgICBkb2NrZXJfcmVwb3NpdG9yeSxcbiAgICAgICAgY29ubmVjdG9yX3ZlcnNpb24sXG4gICAgICAgIGpvYl9zdGF0dXMsXG4gICAgICAgIENPVU5UKDEpIGFzIHN5bmNfY291bnRcbiAgICBGUk9NIG9mZmljaWFsX2Nvbm5lY3Rvcl9zeW5jcyBcbiAgICBHUk9VUCBCWSBjb25uZWN0b3JfZGVmaW5pdGlvbl9pZCwgZG9ja2VyX3JlcG9zaXRvcnksIGNvbm5lY3Rvcl92ZXJzaW9uLCBqb2Jfc3RhdHVzXG4pLFxuc3VjZXNzX2ZhaWx1cmVfYnlfY29ubmVjdG9yX3ZlcnNpb24gQVMgKFxuICAgIFNFTEVDVCBcbiAgICAgICAgY29ubmVjdG9yX2RlZmluaXRpb25faWQsXG4gICAgICAgIGRvY2tlcl9yZXBvc2l0b3J5LFxuICAgICAgICBjb25uZWN0b3JfdmVyc2lvbixcbiAgICAgICAgaWZudWxsKGZhaWxlZCwgMCkgYXMgZmFpbGVkX3N5bmNzX2NvdW50LFxuICAgICAgICBpZm51bGwoc3VjY2VlZGVkLCAwKSBhcyBzdWNjZWVkZWRfc3luY3NfY291bnQsXG4gICAgICAgIGlmbnVsbChzdWNjZWVkZWQsIDApICsgaWZudWxsKGZhaWxlZCwgMCkgYXMgdG90YWxfc3luY3NfY291bnQsXG4gICAgICAgIFNBRkVfRElWSURFKGlmbnVsbChzdWNjZWVkZWQsIDApLCBpZm51bGwoc3VjY2VlZGVkLCAwKSArIGlmbnVsbChmYWlsZWQsIDApKSBhcyBzeW5jX3N1Y2Nlc3NfcmF0ZVxuICAgIEZST00gam9iX3N0YXR1c19wZXJfdmVyc2lvblxuICAgIFBJVk9UXG4gICAgKFxuICAgICAgbWF4KHN5bmNfY291bnQpXG4gICAgICBGT1Igam9iX3N0YXR1cyBpbiAoJ2ZhaWxlZCcsICdzdWNjZWVkZWQnKVxuICAgIClcbilcblNFTEVDVCBcbiAgKiBcbkZST00gXG4gIGFkb3B0aW9uX3Blcl92ZXJzaW9uIFxuICBMRUZUIEpPSU4gc3VjZXNzX2ZhaWx1cmVfYnlfY29ubmVjdG9yX3ZlcnNpb24gXG4gIFVTSU5HIChjb25uZWN0b3JfZGVmaW5pdGlvbl9pZCwgZG9ja2VyX3JlcG9zaXRvcnksIGNvbm5lY3Rvcl92ZXJzaW9uKTsiLCJ0ZW1wbGF0ZS10YWdzIjp7fX0sImRhdGFiYXNlIjoyfSwiZGlzcGxheSI6InRhYmxlIiwicGFyYW1ldGVycyI6W10sInZpc3VhbGl6YXRpb25fc2V0dGluZ3MiOnt9fQ==) for other reviewers to check out